apps/api/src/config/isInTestingMode.ts

```
@@ -0,0 +1 @@
+export const IS_IN_TESTING_MODE = (process.env.IS_TESTING ?? "true") === "true";
```

`export const IS_IN_TESTING_MODE = process.env.IS_TESTING === "true";`

---

And for the file structure. I get the separation of concerns you're trying to achieve but is it really best practice to have one ts file per config var (ex: isInTestingMode)? Seems like it's not scalable at all and maintenance will be really difficult.  

We code have something like.

```
apps/api/src/config/
  index.ts         # central entry point, exports validated config
  env.ts           # env loading + validation schema
  supabase.ts      # supabase-specific config
  auth.ts          # api key / auth config
```
  
 It would fail fast if something is wrong/missing and it'd be easier to navigate/maintain